CRM/Contact/Form/Contact.php

```diff
@@ -375,18 +375,17 @@ public function preProcess() {
   }
 
   private function preProcessLocation() {
-    $this->_addBlockName = CRM_Utils_Request::retrieve('block', 'String');
+    $blockName = CRM_Utils_Request::retrieve('block', 'String');
     $additionalblockCount = CRM_Utils_Request::retrieve('count', 'Positive');
 
     $this->assign('addBlock', FALSE);
-    if ($this->_addBlockName && $additionalblockCount) {
+    if ($blockName && $additionalblockCount) {
       $this->assign('addBlock', TRUE);
-      $this->assign('blockName', $this->_addBlockName);
+      $this->assign('blockName', $blockName);
       $this->assign('blockId', $additionalblockCount);
-      $this->set($this->_addBlockName . '_Block_Count', $additionalblockCount);
+      $this->set($blockName . '_Block_Count', $additionalblockCount);
     }
 
-    $this->assign('blocks', $this->_blocks);
```

Comment:
This was done right before calling this function - see https://github.com/civicrm/civicrm-core/blob/b83d1d71751e4f445a524c4c4a38972a54e45121/CRM/Contact/Form/Contact.php#L308-L307